### DISCUSSION TITLE
is there a way to block a user while signed into an account?

### DISCUSSION THREAD

**Discussion by daxton256** [davidteather/TikTok-Api#1025] [Q&A]
i am making a script that blocks bots on tiktok and i am wondering if there is a way to block users and be signed into an account

**Comment by davidteather** [comment]
Not through this package. But, I think the most to block non-logged in bots you have to make your profile private?

**Comment by mjvieke** [comment]
@mutethecat Did you have any luck making a blocking script? I would love a little demo script that will block all these users 😂...

- You're selected as the winner = **blocked**
- Your selected as the winner = **blocked**
- Youre selected as winner = **blocked**
- YourSelectedWinner = **blocked**
